utils/update_dns.sh

```diff
@@ -3,6 +3,10 @@
 # Update the cloudflare DNS entry to match the public IP address
 # Requires curl, jq, dig(dnsutils)
 
+dig -v 2>/dev/null ; [[ $? == 127 ]] && echo "Error: You need dnsutils to use this tool" && exit 1
```

Comment:
TO-DO A better function that could be in a common library that would check if the tool is missing, based on an arbitrary test command and display the missing packages to get that tool.